trunk/SUAVE/Methods/Aerodynamics/AVL/create_avl_datastructure.py

```diff
@@ -142,22 +143,22 @@ def populate_wing_sections(avl_wing,suave_wing):
                                 #from the leading edge coordinate and not the quarter chord coordinate
                                 if segments[i_segs].sweeps.leading_edge is not None: 
                                         # if leading edge sweep is defined 
-                                        segment_sweep       = segments[i_segs].sweeps.leading_edge * Units.degrees # conversion to radians
+                                        segment_sweep       = segments[i_segs].sweeps.leading_edge  
                                 else:   
                                         # if quarter chord sweep is defined, convert it to leading edge sweep
-                                        sweep_quarter_chord = segments[i_segs].sweeps.quarter_chord * Units.degrees # conversion to radians
```

Comment:
Emilio's comment about using conversion functions here still needs to be addressed.